src/nvim/charset.c

```diff
@@ -1240,7 +1250,7 @@ void getvcol(win_T *wp, pos_T *pos, colnr_T *start, colnr_T *cursor,
           if (enc_utf8 && (c >= 0x80)) {
             incr = utf_ptr2cells(ptr);
           } else {
-            incr = CHARSIZE(c);
+            incr = char_size(c);
```

Comment:
Implicit cast, signed->unsigned, wide->narrow. -Wconversion will not allow this. Though you should just use `int` in function definition.